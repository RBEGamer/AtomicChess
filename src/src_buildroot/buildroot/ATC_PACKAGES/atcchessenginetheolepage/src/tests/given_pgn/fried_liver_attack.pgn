[Event "?"]
[Site "?"]
[Date "????.??.??"]
[Round "?"]
[White "?"]
[Black "?"]
[Result "*"]
[Opening "Fried Liver Attack"]

1. e2-e4 e7-e5 2. Ng1-f3 Nb8-c6 3. Bf1-c4 Ng8-f6 4. Nf3-g5 d7-d5 5. e4xd5
Nf6xd5 6. Ng5xf7 Ke8xf7 *

