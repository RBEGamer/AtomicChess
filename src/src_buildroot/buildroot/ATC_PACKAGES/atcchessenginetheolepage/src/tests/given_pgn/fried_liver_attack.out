[move]	board::PieceType: 5
[move]	board::PieceType: 5
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 3
[move]	board::PieceType: 4
[move]	board::PieceType: 4
[move]	board::PieceType: 5
[move]	board::PieceType: 5
[piece_taken]	 piece: 5 file: 4 rank: 5
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 4 rank: 5
[move]	board::PieceType: 4
[piece_taken]	 piece: 5 file: 6 rank: 7
[move]	board::PieceType: 0
[piece_taken]	 piece: 4 file: 6 rank: 7
